src/d/ast/declaration.d

```diff
@@ -95,13 +95,16 @@ class Module : Declaration {
 
 	Declaration[] declarations;
 
+	Location locationName;
```

Comment:
That is not a good name. Also, group that with the other fields that refers to the same thing, rather than have `declarations` in the middle.